frontend/app/assets/stylesheets/spree/frontend/screen.css.scss

```diff
@@ -359,11 +359,21 @@ nav#taxonomies {
   }
 
   .taxons-list {
-    li {
+    ul {
+      margin-left: 1em;
+    }
+    & > ul {
+      margin-left: 0;
+    }
+    ul li {
+      font-weight: normal;
       a {
         font-size: $main_navigation_font_size
       }
     }
+    .current {
+      font-weight: bold;
+    }
```

Comment:
I think this code can be simplified in this way:

```scss
  ul.taxons-list {
    ul {
      margin-left: 1em;
    }

    li {
      font-size: $main_navigation_font_size;
      font-weight: normal;

      &.current {
        font-weight: bold;
      }
    }
  }
```

Reply:
I agree it can be simplified a little.
However `&.current` is a too specific selector.

Reply:
I think it works with `&.current` since it is applies to `ul.taxons-list li`, which is exactly elements that need this style.

Reply:
@kennyadsl I think @bofrede meant that the selector is too strong in terms of "Do not over specify css selectors" rule of thumb. It would be hard to override this if we over specify selectors. 90% of `!important` usage is because CSS authors uses too specific css selectors.

Reply:
@tvdeyen thanks for your feedback. Yes, it's probably better to have it less specific.